mlir/include/mlir/Interfaces/TilingInterface.td

```diff
@@ -76,11 +112,32 @@ def TilingInterface : OpInterface<"TilingInterface"> {
       >,
       InterfaceMethod<
         /*desc=*/[{
-          Method to return the position of the result tile computed by the tiled operation.
+          Method to return the position of the result tile computed by the
+          tiled operation.
+
+          For operations that return a value (typically a value of type
+          `RankedTensorType`), the generated tiled computation has to also
+          recompute a replacement for the results of the original operation.
+          The tiled implementation of the operation returns a tile of the
+          result(s). This methods returns information about what part of the
+          result tensor is computed by the tiled implementation. The manner in
```

Comment:
```suggestion
          result is computed by the tiled implementation. The manner in
```

Reply:
I think this is expected to mostly work on tensors. For memref-based ops, there are no return values.